src/docfx/docset/Docset.cs

```diff
@@ -168,7 +168,12 @@ internal class Docset
             Repository = repository ?? Repository.Create(DocsetPath, branch: null);
 
             // pass on the command line options to its children
-            _buildScope = new Lazy<HashSet<Document>>(() => CreateBuildScope(Redirections.Files));
+            _buildScope = new Lazy<HashSet<Document>>(() =>
+            {
+                var (errors, scope) = CreateBuildScope(Redirections.Files);
+                report.Write(Config.ConfigFileName, errors);
+                return scope;
+            });
             _redirections = new Lazy<RedirectionMap>(() =>
             {
                 var (errors, map) = RedirectionMap.Create(this);
```

Comment:
consider passing `glob` to `RedirectionMap.Create` and handle redirection related errors in `RedirectionMap.Create`